### PR TITLE
Style cleanup and translations

### PR DESCRIPTION
**Pull request by rockinrobin714**
1. Move hardcoded text to translation file
2. Move all arrow styling / inline styles to sass file
3. Remove console.warn and some other unused code
4. Remove any direction props (we are using `PositionedOverlay` for above/below positioning and will not be including the other directions in MVP)

### CODE REVIEW

File: polaris-react/src/components/InContextLearning/InContextLearning.scss [hunk 1]
```diff
@@ -6,10 +6,39 @@
...
+
+.Arrow {
+  position: absolute;
+  left: calc(25% - var(--p-space-4));
+  border-width: var(--p-space-2);
+  border-style: solid;
+
+  &.above {
+    margin-bottom: var(--p-space-3);
+    bottom: calc(-1 * var(--p-space-4));
```
Comment (rockinrobin714) [inline polaris-react/src/components/InContextLearning/InContextLearning.scss]: is this how we'd use the negative version? I want the bottom to be -20px

Comment (chris-w-mclean-shopify) [reply polaris-react/src/components/InContextLearning/InContextLearning.scss]: This looks fine to me. 👍

**Review by chris-w-mclean-shopify — APPROVED** [review]

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx [hunk 2]
```diff
@@ -111,34 +99,27 @@ export function InContextLearning({onDismiss, title}: Props) {
...
-                    position: 'absolute',
-                    left: 'calc(25% - 1rem)',
-                    borderWidth: '.5rem',
-                    borderStyle: 'solid',
-                    ...verticalMargin,
-                    ...arrowStyles,
-                  }}
-                />
-              )}
+              <div className={arrowClassName} />
```
Comment (alololox) [inline polaris-react/src/components/InContextLearning/InContextLearning.tsx]: nit: can this be a pseudo element?

**Review by alololox — APPROVED** [review]
🤌

**Review by annmariedoyle-shopify — APPROVED** [review]
👍

**Pull request merged**